docker-compose.demo.yml

```diff
@@ -42,15 +42,15 @@ services:
                     - s3.docker.mender.io
         volumes:
             - ./tls.toml:/config/tls.toml
-            - ./certs/api-gateway/cert.crt:/certs/cert.crt
-            - ./certs/api-gateway/private.key:/certs/private.key
-            - ./certs/storage-proxy/cert.crt:/certs/s3.docker.mender.io.crt
-            - ./certs/storage-proxy/private.key:/certs/s3.docker.mender.io.key
+            - ./cert/cert.crt:/certs/cert.crt
+            - ./cert/private.key:/certs/private.key
+            - ./cert/cert.crt:/certs/s3.docker.mender.io.crt
+            - ./cert/private.key:/certs/s3.docker.mender.io.key
```

Comment:
This also needs to be changed [here](https://github.com/mendersoftware/integration/blob/master/production/config/prod.yml.template#L85-L99).